File: apps/web/Article.Portal/Article.Portal/Domain/ArticleSummary.cs
using System;

namespace Article.Portal.Domain
{
    public class ArticleSummary
    {
        public Guid Id { get; set; }

        public DateTime CreationUtcDate { get; set; }

        public DateTime ModificationUtcDate { get; set; }

        public string Author { get; set; }

        public string Title { get; set; }

        public DateTime? PublicationUtcDate { get; set; }

        public string Description { get; set; }
    }
}

File: apps/projector/article-summary-event-elasticsearch-projector/src/domain/articlesummary.ts
import * as moment from "moment";

export class ArticleSummary {
  id: string;
  creationUtcDate: moment.Moment;
  modificationUtcDate: moment.Moment;
  author: string;
  title: string;
  publicationUtcDate: moment.Moment;
  description: string;
}
